djstripe/urls.py

```diff
@@ -26,7 +26,7 @@
 from . import settings as app_settings
 from . import views
 
-
+app_name = 'djstripe'
```

Comment:
Not sure, adding this lets you write `url(r'^stripe/', include('djstripe.urls'))` instead of `url(r'^stripe/', include('djstripe.urls', namespace="djstripe"))` but I meant to separate this commit out from this admin stuff. I'll look into it and do it as a separate PR